package.json

```diff
@@ -1,30 +1,27 @@
 {
-  "name": "@teleporthq/teleport-code-generators",
-  "version": "0.6.0",
+  "name": "root",
+  "workspaces": [
+    "packages/*"
+  ],
+  "private": true,
   "description": "Code generators and UIDL definition library",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
   "scripts": {
-    "start": "tsc -w",
-    "build": "npm run clean && tsc",
-    "clean": "rimraf dist",
     "lint": "tslint -p ./",
+    "build": "lerna run build",
     "test": "jest",
     "test:perf": "jest __tests__/performance",
-    "test:watch": "jest --watch",
     "test:coverage": "npm run test -- --coverage --no-cache",
     "upload:coverage": "codecov",
     "commitizen": "git-cz",
     "prepare": "npm run build",
+    "clean": "lerna run clean",
     "add-contributor": "all-contributors add",
-    "create-react-next": "TS_NODE_FILES=true ts-node --project tsconfig.json ./examples/project-exporters/react-next/index.ts",
-    "create-react-basic": "TS_NODE_FILES=true ts-node --project tsconfig.json ./examples/project-exporters/react-basic/index.ts",
-    "create-vue-basic": "TS_NODE_FILES=true ts-node --project tsconfig.json ./examples/project-exporters/vue-basic/index.ts",
-    "create-vue-nuxt": "TS_NODE_FILES=true ts-node --project tsconfig.json ./examples/project-exporters/vue-nuxt/index.ts"
+    "local:publish": "lerna exec -- npm publish --registry http://localhost:4873",
+    "local:unpublish": "lerna exec -- npm unpublish -f --registry http://localhost:4873"
```

Comment:
Just a reminder, should remove this while merging to master, we should always use ``lerna publish`` for that

Reply:
what if we keep them for the local verdaccio setup?

we could use `npm run publish` for the real npm publish and `npm run publish:local` for the verdaccio server

Reply:
Yeah, that works too :thumbs_up: but the command should be changed. We should use only ``lerna publish`` even for local because in this way each package is packed and published, but the correct way when we are using lerna is ``lerna publish`` which bumps the version and pushes to npm. This was a temp workaround which i made 😅